.github/workflows/ci.yml

```diff
@@ -146,7 +146,7 @@ jobs:
         env:
           PHP_CS_FIXER_IGNORE_ENV: ${{ matrix.PHP_CS_FIXER_IGNORE_ENV }}
           FAST_LINT_TEST_CASES: ${{ matrix.FAST_LINT_TEST_CASES }}
-        run: vendor/bin/phpunit ${{ matrix.phpunit-flags || '--exclude-group auto-review' }}
+        run: vendor/bin/phpunit ${{ matrix.phpunit-flags || '--exclude-group auto-review' }} --testdox
```

Comment:
on my local machine
```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ ./vendor/bin/phpunit --testdox | grep FileRemoval -a10
 ✔ That can report and retrieve invalid errors
 ✔ That can report and retrieve exception errors
 ✔ That can report and retrieve invalid file errors

File Reader (PhpCsFixer\Tests\FileReader)
 ✔ Create singleton
 ✔ Read
 ✔ Read stdin caches
 ✔ Throws exception on fail

File Removal (PhpCsFixer\Tests\FileRemoval)
 ✔ Shutdown removes observed files setup
 ✔ Shutdown removes observed files
 ✔ Clean removes observed files
 ✔ Destruct removes observed files
 ✔ Delete observed file
 ✔ Delete non observed file
 ✔ Sleep
 ✔ Wakeup
```

Reply:
Interesting that it's executed and succeeds 😉. Do you have custom `phpunit.xml` file or do you run tests using dist config? It's failing on my Mac, @kubawerlos verified it on his computer yesterday and had "This test did not perform any assertions" too (with exit code 1).

Reply:
```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ git show HEAD -q
'ee61b7722 2023-06-27 22:23:41 - (HEAD -> master, origin/master, origin/HEAD) bumped version [Dariusz Ruminski]'
```

```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ sw_vers
ProductName:	macOS
ProductVersion:	12.6.2
BuildVersion:	21G320
```

```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ php --version
PHP 8.2.6 (cli) (built: May 11 2023 13:02:10) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.2.6, Copyright (c) Zend Technologies
    with Zend OPcache v8.2.6, Copyright (c), by Zend Technologies
```

```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ composer info -D
composer/semver                                   3.3.2   Semver library that offers utilities, version constraint parsing and validation.
composer/xdebug-handler                           3.0.3   Restarts a process without Xdebug.
doctrine/annotations                              2.0.1   Docblock Annotations Parser
doctrine/lexer                                    3.0.0   PHP Doctrine Lexer parser library that can be used in Top-Down, Recursive Descent Parsers.
justinrainbow/json-schema                         5.2.12  A library to validate a json schema.
keradus/cli-executor                              v2.0.0  CLI execution wrapper
mikey179/vfsstream                                v1.6.11 Virtual file system to mock the real file system in unit tests.
php-coveralls/php-coveralls                       v2.5.3  PHP client library for Coveralls API
php-cs-fixer/accessible-object                    v1.1.0  A library to reveal object internals.
php-cs-fixer/phpunit-constraint-isidenticalstring v1.3.0  Constraint for testing strings considering not-same line endings.
php-cs-fixer/phpunit-constraint-xmlmatchesxsd     v1.3.0  Constraint for testing XML against XSD.
phpspec/prophecy                                  v1.17.0 Highly opinionated mocking framework for PHP 5.3+
phpspec/prophecy-phpunit                          v2.0.2  Integrating the Prophecy mocking library in PHPUnit test cases
phpunit/phpunit                                   9.6.9   The PHP Unit Testing framework.
phpunitgoodpractices/polyfill                     v1.7.0  Lacking future-compat polyfills for PHPUnit.
phpunitgoodpractices/traits                       v1.9.2  Highly opinionated PHPUnit good practices enforcer.
sebastian/diff                                    4.0.5   Diff implementation
symfony/console                                   v6.3.0  Eases the creation of beautiful and testable command line interfaces
symfony/event-dispatcher                          v6.3.0  Provides tools that allow your application components to communicate with each other by dispatching events and listening to them
symfony/filesystem                                v6.3.1  Provides basic utilities for the filesystem
symfony/finder                                    v6.3.0  Finds files and directories via an intuitive fluent interface
symfony/options-resolver                          v6.3.0  Provides an improved replacement for the array_replace PHP function
symfony/phpunit-bridge                            v6.3.1  Provides utilities for PHPUnit, especially user deprecation notices management
symfony/polyfill-mbstring                         v1.27.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php80                            v1.27.0 Symfony polyfill backporting some PHP 8.0+ features to lower PHP versions
symfony/polyfill-php81                            v1.27.0 Symfony polyfill backporting some PHP 8.1+ features to lower PHP versions
symfony/process                                   v6.3.0  Executes commands in sub-processes
symfony/stopwatch                                 v6.3.0  Provides a way to profile code
symfony/yaml                                      v6.3.0  Loads and dumps YAML files
```

```
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
ker@d ~/github/FriendsOfPHP/PHP-CS-Fixer λ
```

Reply:
OK, it seems like `FileRemovalTest` is succeeding when run in full suite, but fails when run separately (either via IDE or CLI arg). I suppose that's why it fails in ParaUnit too.